Game/Game.js

```diff
@@ -1,45 +1,63 @@
 class Game {
-    constructor(wordToGuess, maxWrongAttempts) {
+    constructor(wordToGuess, maxWrongAttempts = 6) {
         // this.alive = true;
         this.player1;
         this.player2;
-        this.wordToGuess = wordToGuess;
+        this.wordToGuess = wordToGuess.toLowerCase();
         this.maxWrongAttempts = maxWrongAttempts;
         this.attemptedGuesses = []; // All of these will be displayed as guesses
         this.wrongGuesses = 0;
         this.correctGuesses = []; // These will be displayed as correct guesses only
     }
+
     reviewAttempt(guess) {
+        guess = guess.toLowerCase();
+        if (this.attemptedGuesses.includes(guess)) {
+          return;
+        }
         this.attemptedGuesses.push(guess);
         if (this.wordToGuess != guess && !this.wordToGuess.includes(guess)) {
-            this.wrongGuesses += 1;
-            this.wrongAttempt();
+            return this.wrongAttempt(guess);
         } else {
             this.correctGuesses.push(guess);
-            this.correctAttempt();
+            return this.correctAttempt();
         }
     }
-    wrongAttempt() {
+
+    wrongAttempt(guess) {
+        this.wrongGuesses += 1;
         if (this.wrongGuesses >= this.maxWrongAttempts) {
-            alert('The man is dead');
-            location.reload();
+            return 'The man is dead';
         }
+        return `'${guess}' was a bad guess.`
     }
+
     correctAttempt() {
         let theWordSplit = this.wordToGuess.split('');
         if (this.correctGuesses.slice(-1)[0] == this.wordToGuess) {
-            winGame();
+            return this.winGame();
         }
-        if (this.correctGuesses.includes(theWordSplit)) {
-            winGame();
+        if (theWordSplit.every(letter => this.correctGuesses.includes(letter))) {
+            return this.winGame();
         }
     }
+
     winGame() {
-        alert('The man has survived');
-        location.reload();
+        return 'The man is alive';
+    }
```

Comment:
Actually, piggybacking on the previous comment, I know realize we might want to have more functionality to this.winGame(), so maybe we can keep it above, if that is the case here.

Reply:
Oh lol! We both just did the same thing hahaha.